src/PlusServer/vtkPlusOpenIGTLinkServer.cxx

```diff
@@ -47,22 +51,25 @@ See License.txt for details.
   #include "vtkPlusOpenIGTLinkServerLinux.cxx"
 #endif
 
-static const double DELAY_ON_SENDING_ERROR_SEC = 0.02;
-static const double DELAY_ON_NO_NEW_FRAMES_SEC = 0.005;
-static const int NUMBER_OF_RECENT_COMMAND_IDS_STORED = 10;
-static const int IGTL_EMPTY_DATA_SIZE = -1;
-
-const float vtkPlusOpenIGTLinkServer::CLIENT_SOCKET_TIMEOUT_SEC = 0.5;
+namespace

```

Comment:
If you use anonymous namespace then there is no need for static to make the variables available only in the current translation unit. You can remove the static keywords.

Reply:
I thought so, I wasn't sure. Done.